website/blog/2021-04-12-funding-update.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Babel Funding Update"
+author: Babel Core Team
+date: 2021-04-12 0:00:00
+categories: announcements
+share_text: "Babel Funding Update"
+---
+
+In March 2018 [Henry](https://twitter.com/left_pad) started an [experiment](https://www.henryzoo.com/in-pursuit-of-open-source-part-1/): would it be possible to work full-time on Babel, entirely sustained by donations from Babel users?
+
+In November 2019, after successfully guaranteeing a salary to a maintainer for over a year, we [expanded our goal](https://babeljs.io/blog/2019/11/08/babels-funding-plans): would it also be possible to support three additional part-time maintainers ([Jùnliàng](https://twitter.com/JLHwung), [Kai](https://twitter.com/kai_cataldo) and [Nicolò](https://twitter.com/NicoloRibaudo))?
+
+Nicolò, Jùnliàng and Henry are still paid to work on Babel, and together with them there is a talented team that helps after or during their day-to-day job.
+
+<!-- truncate -->
+
+## Looking back
+
+After a few months being paid to work on Babel, Henry started receiving $11,000 USD per month. The goal was to establish the number as a baseline salary for working full-time on an open source project that seemed reasonable given the donations at the time.
+
+When we added three new paid maintainers, we couldn't afford to pay everyone the full $11,000 USD rate: Jùnliàng, Nicolò and Kai started at $2,000 USD per month, with the idea of increasing that amount when we would have received more donations.
+
+It has been more than one year since then and in the meanwhile Kai stepped down, but we still didn't have a chance to reconsider how much everyone is being paid.
+
+We have a hard time justifying the pay differences, and there isn't an obvious reason anymore for which someone should be paid more than the other team members:
+
+- We want to focus on ourselves as _people_, not as employees of a hypothetical company. We don't want to track meticulously how many hours we work, or how productive we are.
+- It's not obvious what we would consider as "working time". Part of what we do is to collaborate with other projects, but what exactly is the line between something done for Babel and something done as a personal contribution to the JavaScript ecosystem?
+- It's not possible to exactly measure how much we "produce": not every commit takes the same effort, not every task has the same impact, not every experiment has a direct result.
+
+For these reasons, we'll now change the current situation.
+
+## The future
+
+Starting from April, Henry, Nicolò and Jùnliàng will all be paid the same amount.
+
+This means we had to make a choice: does "pay everyone the same" mean that we should redistribute the total amount we spend monthly equally between us, or that we should bring everyone up to the level of who was being paid the most?
```

Comment:
> "pay everyone the same" mean that we should redistribute the total amount we spend monthly equally between us,

Given that the project does have historical precedent for leveling the comp field for past contributions
https://opencollective.com/babel/expenses/9074

Have y'all considered doing this for others on the project for their prior work?

Reply:
No, we have only decided to change how we are being paid from now on. We can write it explicitly.